.JuliaFormatter.toml

```diff
@@ -1,3 +1,4 @@
 remove_extra_newlines = true
 always_for_in = true
 conditional_to_if = true
+ignored = ["bin", "docs"]
```

Comment:
I'm totally fine with these being ignored but I'm just wondering things are still fine if they are not ignored right?

Reply:
I guess. Can't remember why I picked those specifically. They should probably not be ignored.